src/components/MultiChoice/index.test.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import { act, render, screen, within } from '@testing-library/react';
+
+import { answerFabricator } from 'tests/fabricator';
+import { Answer } from 'types/answer';
+
+import MultiChoice, { multiChoiceDataTestIds } from '.';
+
+describe('MultiChoice', () => {
+  const answers = answerFabricator.times(3);
+
+  it('renders a multi choice component', () => {
+    const onValuesChanged = () => {
+      // Do nothing
+    };
+    render(<MultiChoice items={answers} isPickOne={false} onValuesChanged={onValuesChanged} />);
```

Comment:
If the onValuesChanged doesn't do anything, do we need a separate function for it?
```suggestion
    const onValuesChanged = () => {
      // Do nothing
    };
    render(<MultiChoice items={answers} isPickOne={false} onValuesChanged={() => {}} />);
```
But do you think `onValuesChanged` should be required?

Reply:
@hoangmirs, I followed @nvminhtue's suggestion to make `onValuesChanged` required. This is because `onValuesChanged` is always needed when rendering `MultiChoice` in UI components. 🙏

Reply:
@hoangmirs updated in [9cf9cc7](https://github.com/manh-t/react-survey/pull/79/commits/9cf9cc7e64b19e65e275327d6b0981dc4379edc3)